corehq/apps/locations/fixtures.py

```diff
@@ -256,16 +255,16 @@ def get_location_fixture_queryset(user):
 
     user_location_ids = list(user_locations.order_by().values_list("id", flat=True))
 
-    return _location_queryset_helper(user.domain, user_location_ids)
+    return get_location_fixture_queryset(user.domain, user_location_ids)
 
 
-def _location_queryset_helper(domain, location_pks):
+def get_location_fixture_queryset(domain, location_pks, case_sync_restriction=False):
```

Comment:
I don't have any great suggestions to offer, but `get_location_fixture_queryset_for_user` and `get_location_fixture_queryset` sound very similar and it's kinda hard to tell when to use one vs the other

Reply:
I think the ideal situation would be to come up with a better name for what happens in the SQL function.  Maybe Daniel has ideas?  It was previously used just to generate the location fixture, but now that's expanding.  Maybe `get_section_of_location_tree` or something?  I dunno, it's a hard one.